fix Emojistyle type

tylim88 opened the pull request:
type error
![image](https://github.com/ealush/emoji-picker-react/assets/5227509/a6b4e517-0a10-40c8-aca7-ac1fd48e5175)

fix:
![image](https://github.com/ealush/emoji-picker-react/assets/5227509/611919fa-41c3-4d9d-8891-d66a186afce3)

advise: don't use enum

ealush commented:
Technically I am fine with this, but seems like it fails the build (yarn build).

![image](https://github.com/ealush/emoji-picker-react/assets/11255103/4d064e56-a683-4172-8d6a-14536b30b0f5)
(It shows the error for the onEmojiClick, but I can confirm that it only happens with your change)

I don't have the time to debug this, but feel free to do any of the following:
1. Replace the Enum in the config with the string type (keep the enum elsewhere, because it is exported, and I don't want to change the existing exposed interface)
2. Find why it is breaking in the build step and fix it. I currently don't have the time to do it.

Either of these is fine with me, I'd personally go with 1. because, as you noted, enums aren't great.

tylim88 commented:
this is because the TS version of tsdx is too old

tsdx seem dead

I replaced tsdx with dts-cli (a fork of tsdx)

I fixed all EmojiStyle type errors, the runtime code remain intact
